README.md

```diff
@@ -1,41 +1,41 @@
 
-# Firehose-to-Logdna
+# Firehose-to-LogDNA
 
-This is a Nozzle to drain log from firehose in CloudFoundry to LogDNA.
+This is a Nozzle to send firehose events from CloudFoundry to LogDNA.
 
 Fork From: https://github.com/cloudfoundry-community/firehose-to-syslog
 
 # Usage
 
-Currently the Nozzle only run as an app in Cloud Foundry.
+Currently the Nozzle only runs as an app in Cloud Foundry.
 
-1 - Download the latest release of Firehose-to-Logdna.
+1. Download the latest release of Firehose-to-LogDNA.
 ```
-    $git clone https://github.com/logdna/logdna-cloudfoundry.git
-    $cd firehose-to-syslog
+    git clone https://github.com/logdna/logdna-cloudfoundry.git
```

Comment:
are these 4 trailing spaces needed in front of commands?

Reply:
I think you mean leading spaces, and I don't know. Let me follow the same format as https://github.com/cloudfoundry-community/firehose-to-syslog/blob/develop/README.md.

Reply:
this looks better